.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 rvm:
   - '2.4'
   - '2.3'
+  - '2.5'
```

Comment:
I'd keep the list sorted and newest first. They get tested first and IMHO the newest version should be tested first.